packages/suite/src/components/suite/NavigationBar/components/NavigationActions/components/NotificationsDropdown/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { Translation, Notifications } from '@suite-components';
+import { Dropdown, DropdownRef, variables } from '@trezor/components';
+import React, { useRef, useCallback, useState } from 'react';
+import styled from 'styled-components';
+import ActionItem from '../ActionItem';
+import { useActions } from '@suite-hooks';
+import * as notificationActions from '@suite-actions/notificationActions';
+
+const Wrapper = styled.div`
+    display: flex;
+    z-index: 10;
```

Comment:
Does the z-index have to be exactly `10` to fit in the z-index hierarchy? Isn't 1 enough? If it has to be higher due to the hierarchy let's consider putting it in a global z-index layer config.